bin/application

```diff
@@ -25,8 +25,8 @@ SetWorkerDatabasePoolSize(){
 }
 
 WaitForPostgres () {
-  export PGPASSWORD="$DB_ENV_POSTGRES_PASSWORD"
-  until psql -h "$DB_PORT_5432_TCP_ADDR" -p "$DB_PORT_5432_TCP_PORT" -U "$DB_ENV_POSTGRES_USER" -c '\l' &> /dev/null; do
+  export PGPASSWORD="$POSTGRES_PASSWORD"
+  until psql -h "$POSTGRES_HOST" -p "$POSTGRES_PORT" -U "$POSTGRES_USER" -c '\l' &> /dev/null; do
```

Comment:
Okay, so we'll need the Infrastructure/ParameterStore changes before this.  But you were expecting that.

Reply:
yes, and I merged those earlier today, so we should be good.